src/providers/Celo.ts

```diff
@@ -211,6 +214,23 @@ export async function getCurveUSDC(): Promise<ProviderResult> {
   }
 }
 
+export async function getUniV3Holdings(
+  address: string
+): Promise<ProviderResult<Map<string, BigNumber>>> {
+  try {
+    const uniV3Holdings = await uniV3BalanceCalculator.calculateUniV3PoolBalance(address)
+    if (uniV3Holdings.has(STAKED_CELO_ERC20_ADDRESS)) {
+      uniV3Holdings.set(
+        STAKED_CELO_ERC20_ADDRESS,
+        await StakedCeloProvider.Instance.stCeloToCelo(uniV3Holdings.get(STAKED_CELO_ERC20_ADDRESS))
```

Comment:
Yeah let's move this conversion further down the line as discussed before and call the StakedCeloProvider from there, it's fine because it's a singleton.